owl/webapp.py

```diff
@@ -1294,7 +1294,13 @@ def main():
         app = create_ui()
 
         app.queue()
-        app.launch(share=False, favicon_path="../assets/owl-favicon.ico")
+        launch_kwargs = {
+            "share": False,
+            "favicon_path": "../assets/owl-favicon.ico"
+        }
+        if os.getenv("OWL_SERVER_NAME") is not None:
+            launch_kwargs["server_name"] = os.getenv("OWL_SERVER_NAME")
```

Comment:
OWL_SERVER_NAME should be validated to confirm it is a valid hostname or IP address